products/firestore/api.yaml

```diff
@@ -108,3 +108,80 @@ objects:
                 be specified.
               values:
                 - :CONTAINS
+  - !ruby/object:Api::Resource
+    name: 'Document'
+    base_url: projects/{{project}}/databases/{{database}}/documents/{{collection}}
+    create_url: projects/{{project}}/databases/{{database}}/documents/{{collection}}?documentId={{document_id}}
+    update_verb: :PATCH
```

Comment:
Can you add a test that performs an update?

Reply:
will do !